files/__addonLocation__/babel.config.cjs

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  "plugins": [
+<% if (typescript) { %>  ["@babel/plugin-transform-typescript", { "allExtensions": true, "onlyRemoveTypeImports": true, "allowDeclareFields": true }],
+<% } %>    "@embroider/addon-dev/template-colocation-plugin",
+    ["decorator-transform", {
+      runtime: {
+        import: require.resolve('decorator-transforms/runtime'),
```

Comment:
If you're planning to ship with `decorator-transforms`as a runtime dependency (which is reasonable), then this should be just `import: 'decorator-transform/runtime'` with no require.resolve, because the addon should be able to resolve that name normally.

Reply:
excellent!